packages/server/graphql/types/GitLabIntegration.ts

```diff
@@ -87,14 +89,14 @@ const GitLabIntegration = new GraphQLObjectType<any, GQLContext>({
         return fetchGitLabProjects(teamId, userId, context, info)
       }
     },
-    projectIssues: {
+    projectsIssues: {
       type: new GraphQLNonNull(GitLabProjectIssuesConnection),
       args: {
         first: {
           type: GraphQLNonNull(GraphQLInt)
         },
         after: {
-          type: GraphQLISO8601Type,
+          type: GraphQLString,
```

Comment:
@mattkrick if I change the `after` and `cursor` type to `GraphQLList(GraphQLNonNull(GitLabProjectIssuesCursorDetailInput)`, I get the following error: `The type of GitLabIntegrationEdge.cursor must be Output Type but got: [GitLabProjectIssuesCursorDetailInput!]`

Relay requires cursors to be a string ([see here](https://relay.dev/graphql/connections.htm#sec-Cursor)), so if we create type `GitLabProjectIssuesCursorDetailInput`, won't we still need to stringify & parse the cursors each time and therefore not get any additional type safety?

Reply:
whoa, TIL!
you could create a custom scalar that accepts a string & validates the shape of the stringified JSON, or you could do that validation within the resolve function. the latter is probably better since I don't imagine we'll need to repeat this in the near term. we just need some validation to make sure the JSON looks like `{after, fullPath][]`.

Reply:
Makes sense! I've updated the JSON structure and I'm validating the structure in the resolver here: https://github.com/ParabolInc/parabol/blob/155e2a996117fa7648f50a896298fb3766c7421e/packages/server/graphql/types/GitLabIntegration.ts#L157. Is that what you had in mind?